client/src/pages/style.css

```diff
@@ -0,0 +1,61 @@
+label {
+    margin: 0 5px;
+}
+
+.create-page {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    margin-top: 25px;
```

Comment:
Minor suggestion: Since we have multiples pages, it'd probably be easier if we do 25px margin bottom to the Navbar instead? Please feel free to not do it if you prefer otherwise